README.md

```diff
@@ -138,6 +138,9 @@ jobs:
           script: ./gradlew connectedCheck
 ```
 
+Note: if you want to use [gradle-build-action](https://github.com/gradle/gradle-build-action) with `reactivecircus/android-emulator-runner`,
+need to declare `uses: gradle/gradle-build-actionv2` without any arguments before `uses: reactivecircus/android-emulator-runner@v2`, more details see [#229](https://github.com/ReactiveCircus/android-emulator-runner/issues/229).
+
```

Comment:
This isn't specific to this action, so I think we can just replace the `actions/cache@v2` step in the snippet above with `uses: gradle/gradle-build-actionv2`.

Reply:
4fae9cef9371fa74720173ed7919bc20a279476d

Reply:
<img width="886" alt="image" src="https://user-images.githubusercontent.com/10363352/156705164-9b70d513-f426-4da6-a17f-5dda13273549.png">